local/scripts/deploy.sh

```diff
@@ -56,13 +56,15 @@ export GIT_WORK_TREE="$UPSTREAM_DIR"
 rm -rf "$UPSTREAM_DIR"
 
 git clone "$UPSTREAM_REPO" "$UPSTREAM_DIR/.git"
-git checkout -B "$UPSTREAM_BRANCH"
+
+# Checkout the release branch or create it if it doesn't exist.
+git checkout "$UPSTREAM_BRANCH" || git checkout -B "$UPSTREAM_BRANCH"
```

Comment:
@delawski This fixes the issue with the deploys saying that the upstream has diverted. Previously it was essentially creating a new branch with the name of the upstream branch and it was obviously out of sync with the upstream.